assets/pages/weekly-subscription-landing/components/weeklyForm.jsx

```diff
@@ -3,25 +3,66 @@
 // ----- Imports ----- //
 
 import React from 'react';
+import { connect } from 'react-redux';
 
-import uuidv4 from 'uuid';
+import SvgInfo from 'components/svgs/information';
 
 import WeeklyCta from './weeklyCta';
+import { subscriptions, type Subscription } from './../weeklySubscriptionLandingReducer';
+import { type State } from './../weeklySubscriptionLandingReducer';
+import WeeklyFormLabel from './weeklyFormLabel';
+
+// ---- Types ----- //
+
+type PropTypes = {|
+  checked?: ?Subscription,
+|};
 
 // ----- Render ----- //
 
-const FormLabel = ({ type }: {type: string}) => {
-  const id = uuidv4();
-  return (<label htmlFor={id}><input id={id} type="radio" name="sub-type" value={type} />{type}</label>);
+const onSubmit = (ev: Event, subscription: ?Subscription) => {
+  ev.preventDefault();
+  if (subscription) {
+    console.log(`now leaving to the ${subscription} checkout 🚀`);
```

Comment:
Is this still needed?

Reply:
Good shout! I need to get a list of URLs to send users to, who would be the best person to ask?